src/objectTranslations/Address__c-ca.objectTranslation

```diff
@@ -1,481 +1,487 @@
-      <translation>28</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>29</masterLabel>
-      <translation>29</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>30</masterLabel>
-      <translation>30</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>31</masterLabel>
-      <translation>31</translation>
-    </picklistValues>
-  </fields>
-  <fields>
-    <name>Seasonal_Start_Month__c</name>
-    <help>El mes de l'any en què aquesta adreça substitueix l’adreça predeterminada.</help>
-    <label>Mes d'inici estacional</label>
-    <picklistValues>
-      <masterLabel>1</masterLabel>
-      <translation>1</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>2</masterLabel>
-      <translation>2</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>3</masterLabel>
-      <translation>3</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>4</masterLabel>
-      <translation>4</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>5</masterLabel>
-      <translation>5</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>6</masterLabel>
-      <translation>6</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>7</masterLabel>
-      <translation>7</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>8</masterLabel>
-      <translation>8</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>9</masterLabel>
-      <translation>9</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>10</masterLabel>
-      <translation>10</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>11</masterLabel>
-      <translation>11</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>12</masterLabel>
-      <translation>12</translation>
-    </picklistValues>
-  </fields>
-  <fields>
-    <name>Seasonal_Start_Year__c</name>
-    <help>L'any en què aquesta adreça substitueix l’adreça predeterminada.</help>
-    <label>Any d'inici estacional</label>
-  </fields>
-  <layouts>
-    <layout>HEDA Address Layout</layout>
-    <sections>
-      <label>Enllaços personalitzats</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
-  <nameFieldLabel>ID d’adreça</nameFieldLabel>
-  <standardFields>
-    <name>Name</name>
-    <label>ID d’adreça</label>
-  </standardFields>
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- Address --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- Address --></value>
+    </caseValues>
+    <fields>
+        <help>Seleccioneu quin tipus d’adreça es tracta, per exemple, de casa, de la feina, de la residència de vacances o d’una altra.</help>
+        <label>Tipus d&apos;adreça</label>
```

Comment:
@MatthewBlanski is this change ok in the label?

Reply:
Replacement of apostrophes with apos are acceptable. This is a platform function.